x-pack/plugins/ml/public/cases/register_cases_attachments.ts

```diff
@@ -8,14 +8,17 @@
 import type { CasesPublicSetup } from '@kbn/cases-plugin/public';
 import type { CoreStart } from '@kbn/core/public';
 import { registerAnomalyChartsCasesAttachment } from './register_anomaly_charts_attachment';
+import { registerSingleMetricViewerCasesAttachment } from './register_single_metric_viewer_attachment';
 import type { MlStartDependencies } from '../plugin';
 import { registerAnomalySwimLaneCasesAttachment } from './register_anomaly_swim_lane_attachment';
 
 export function registerCasesAttachments(
   cases: CasesPublicSetup,
   coreStart: CoreStart,
-  pluginStart: MlStartDependencies
+  pluginStart: MlStartDependencies,
+  mlServices?: any
```

Comment:
should it be `SingleMetricViewerServices` instead?

Reply:
Updated with name change in 6c61014b8a7ef4a3167a642e5f72f1938ea25c50